Projects/My_website/myWebsite/account/serializers.py

```diff
@@ -24,6 +25,11 @@ def validate(self, data):
             if User.objects.filter(email=data["email"]).exists():
                 raise serializers.ValidationError("email is exist")
 
+        if data["password"]:
+            status, message = StrongPasswordValidator(data["password"])
```

Comment:
@codal-uchoudhary can you please use `is_valid` instead of status, as it makes more sense?